roles/pulp3/tasks/service.yml

```diff
@@ -11,7 +10,10 @@
     - pulp_web
     - pulp_resource_manager
     - pulp_worker@
-  notify: 'reload systemd and restart {{ item }}'
```

Comment:
This needs to remain to reload and restart services that have changed.

Reply:
Notify doesn't run on call, but at the end of block execution, which cause the test to fail.
That's why I've added line 14 (see my comment there)